docker-for-mac/index.md

```diff
@@ -512,6 +494,9 @@ Check out these [Docker Cloud topics](/docker-cloud/index.md) to learn more:
 * [Builds and Images](/docker-cloud/builds/index.md)
 * [Swarms in Docker Cloud](/docker-cloud/cloud-swarm/index.md)
 
+Also, here is a direct link to the [Docker Cloud
```

Comment:
I'd consider rewording this to something like "You can also go directly to <link URL as link text>." to make it more concise.